src/Monitor/DataCollectionRule.Autorest/docs/New-AzDataCollectionEndpoint.md

```diff
@@ -8,16 +8,16 @@ schema: 2.0.0
 # New-AzDataCollectionEndpoint
 
 ## SYNOPSIS
-Create a data collection endpoint.
+create a data collection endpoint.
```

Comment:
The first letter is updated with lower case for all the changes.

Reply:
The issue here is due to a problem caused by an update to autorest tool, and Beisi is in the process of modifying the code that is causing the issue.

Reply:
Please manually update them all for now.